### PR TITLE
fix: rh-alert not using correct font-family, cursor pointer

### PR DESCRIPTION
**Pull request by brianferry**
Fixes issues related to font-family and cursor pointer for rh-alert described https://github.com/RedHat-UX/red-hat-design-system/issues/516

### CODE REVIEW

File: elements/rh-alert/rh-alert.css [hunk 1]
```diff
@@ -73,6 +75,8 @@ header {
 
 header ::slotted(:is(h1,h2,h3,h4,h5,h6)) {
   margin: 0;
+
+  --rh-font-family-heading: var(--_font-family);
```
Comment (marionnegp) [inline elements/rh-alert/rh-alert.css:79]: The XD kit's version is showing that text in the header should be using Red Hat Text and --rh-font-size-body-text-sm. Font weight's correct.

File: elements/rh-alert/rh-alert.css [hunk 2]
```diff
@@ -52,6 +53,7 @@
   display: grid;
   grid-template-columns: min-content 1fr;
   gap: var(--rh-space-xs, 4px);
+  font-family: var(--_font-family);
 }
```
Comment (marionnegp) [inline elements/rh-alert/rh-alert.css:57]: Border width for the container should be --rh-border-width-md for all variants

File: elements/rh-alert/rh-alert.css [hunk 3]
```diff
@@ -123,15 +127,21 @@ footer ::slotted([slot="actions"]) {
...
 
+footer ::slotted([slot="actions"]:hover) {
+  cursor: pointer;
+  text-decoration: underline;
+  color: var(--rh-color-interactive-blue-darkest, #004080);
+}
+
 :host(:not([variant])) #container,
 :host([toast]) #container {
```
Comment (marionnegp) [inline elements/rh-alert/rh-alert.css:146]: I believe the token for the box shadow on the toast variant should be --rh-box-shadow-lg. That's the closest to the design specs.

**Review by marionnegp — COMMENTED** [review]
The changes you made today look good. I found a few things from the previous iteration of rh-alert that weren't consistent with what we have in XD.

File: elements/rh-alert/rh-alert.css (hunk 1 above)
Comment (brianferry) [reply elements/rh-alert/rh-alert.css:80]: Updated the font size for the header, looks like the font-family was already red hat text?

File: elements/rh-alert/rh-alert.css (hunk 2 above)
Comment (brianferry) [reply elements/rh-alert/rh-alert.css:57]: Border width has been updated ✅

File: elements/rh-alert/rh-alert.css (hunk 3 above)
Comment (brianferry) [reply elements/rh-alert/rh-alert.css:151]: Updated this to --rh-box-shadow-lg w/ the token values

**Comment by brianferry** [comment]
@marionnegp - Also noticed the margin between the header & the description looked off compared to the UX docs.  In the UX docs it looks like the description is 8px away from the header and 16px away from the action links so I have updated those in the demo as well to use the relevant tokens.

**Comment by marionnegp** [comment]
Awesome, @brianferry! The changes look good to me.

Does it matter that the component width changed in the demo?
Screen Shot 2022-09-09 at 1 23 59 PM

**Comment by brianferry** [comment]
@marionnegp - That's a good question, so in the mockups I see it's 396px width and in the component itself we set a max width of 40vh which probably isn't what we want from the demos.  Would we want to set the width to 396px regardless of the content inside so that it always appears the same?  If we set it to a max-width of 396px then we could end up with something like this when we have less content

image

**Comment by marionnegp** [comment]
@brianferry, only the toast variant should have a max-width of 555px. The other two variants don't need a max-width.

**Comment by brianferry** [comment]
@marionnegp - Got it, I was incorrect in thinking that all the rh-alerts had a max-width of 40vh, that was a demo specific css and not indicative of the component itself so I removed it.  I also added the max-width for the toast variant.

**Comment by marionnegp** [comment]
No prob! I wasn't sure at first either, @brianferry.

**Review by marionnegp — COMMENTED** [review]
This looks good to me, but I'll let you go ahead and merge it!

**Review by marionnegp — APPROVED** [review]

**Pull request merged**